tests/conftest.py

```diff
@@ -94,7 +96,25 @@ def app():
 def client(app):
     return TestClient(app)
 
+@pytest.fixture
+def image_path_jpeg():
+	image = "cytomine-org-logo.jpeg"
+	path = "/data/pims/upload_test_jpeg/"
+	return [path, image]
 
+@pytest.fixture
+def image_path_png():
+	image = "cytomine-org-logo.png"
+	path = "/data/pims/upload_test_png/"
+	return [path, image]
+
+@pytest.fixture(scope='module')
+def image_path_tiff():
+	image = "earthworm-transv-posterior-to-clitellum-02.tiff"
```

Comment:
Is it a planar or pyramidal TIFF ? This are not the same format in PIMS world :/. Could be great to strictly follow format names, e.g.:
* A fixture `image_path_planartiff()` for `PlanarTiffFormat`
* A fixture `image_path_pyrtiff()` for `PyrTiffFormat`
The same applies for test names.